src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -1493,7 +1489,7 @@ public static boolean hasUsableWrench(EntityPlayer player, BlockPos pos)
 				return true;
 			}
 			
-			if(isCoFHHammer(tool.getItem()) && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
+			if(isCoFHHammer(tool.getItem())) // TODO Implement CoFH Hammer && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
```

Comment:
If IToolHammer isn't in the RedstoneFlux api, I'd say just leave that one class in `src/java/api`; as long as you don't have a copy of `package-info` and the `@Optional` is present on the Mek wrench it should all be fine